Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/BlobTriggerCSharp.cs

```diff
@@ -1,23 +1,25 @@
-using System;
 using System.IO;
+using System.Threading.Tasks;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
 namespace Company.Function
 {
     public class BlobTriggerCSharp
     {
-        private readonly ILogger _logger;
+        private readonly ILogger<BlobTriggerCSharp> _logger;
 
-        public BlobTriggerCSharp(ILoggerFactory loggerFactory)
+        public BlobTriggerCSharp(ILogger<BlobTriggerCSharp> logger)
         {
-            _logger = loggerFactory.CreateLogger<BlobTriggerCSharp>();
+            _logger = logger;
         }
 
-        [Function("BlobTriggerCSharp")]
-        public void Run([BlobTrigger("PathValue/{name}", Connection = "ConnectionValue")] string myBlob, string name)
+        [Function(nameof(BlobTriggerCSharp))]
+        public async Task Run(
+            [BlobTrigger("PathValue", Connection = "ConnectionValue")] Stream stream)
         {
-            _logger.LogInformation($"C# Blob trigger function Processed blob\n Name: {name} \n Data: {myBlob}");
+            using var blobStreamReader = new StreamReader(stream);
+            _logger.LogInformation("Blob content: {content}", await blobStreamReader.ReadToEndAsync());
```

Comment:
might be nice to assign this to a value for easier readability

`var content = await blobStreamReader.ReadToEndAsync()`

Reply:
and `_logger.LogInformation($"C# Blob trigger function Processed blob\n Name: {name} \n Data: {content}");`